au4/src/trackedit/internal/au3/au3interaction.cpp

```diff
@@ -109,6 +109,13 @@ bool Au3Interaction::canPasteClips(const std::vector<TrackId>& dstTracksIds, sec
             //! TODO AU4: show dialog
```

Comment:
There is LOGDA above, please replace it with LOGD (LOGDA should be used only for current debugging of current work)